Gemfile

```
@@ -3,7 +3,7 @@ FOREMAN_GEMFILE = __FILE__ unless defined? FOREMAN_GEMFILE
 
 source 'https://rubygems.org'
 
-gem 'rails', '~> 6.1.6.1'
+gem 'rails', '~> 6.1'
```

This will also allow 6.2, which we do not want.
I think it should be either
```suggestion
gem 'rails', '~> 6.1.6'
```
or
```suggestion
gem 'rails', '>= 6.1.1.6', '<6.2'
```